addons/hr_expense/static/tests/tours/expense_upload_tours.js

```diff
@@ -76,3 +76,29 @@
             }
         },
     ]});
+
+    registry.category("web_tour.tours").add('hr_expense_access_rights_test_tour', {
+        test: true,
+        url: "/web",
+        steps: () => [stepUtils.showAppsMenuItem(),
+        {
+            content: "Go to Expense",
+            trigger: '.o_app[data-menu-xmlid="hr_expense.menu_hr_expense_root"]',
+        },
+        {
+            content: "Go to My Expenses",
+            trigger: 'button[data-menu-xmlid="hr_expense.menu_hr_expense_my_expenses"]',
+        },
+        {
+            content: "Go to My Expenses to Report",
```

Comment:
```suggestion
            content: "Go to My Reports",
```

Reply:
Thanks, I always forget to double checks comments